<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NAS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../resources/icons/AppIcon.jpeg</normaloff>../../resources/icons/AppIcon.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 238, 226);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0 y1:0, x2:0 y2:1, stop:0 #373a3c, stop:1 #72777d);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="Header" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>151</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>151</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #373a3c;
border-bottom: 1px solid white;


</string>
      </property>
      <layout class="QHBoxLayout" name="Header_layout">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="fit_icon">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../resources/icons/FitIcon.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="header_info_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="app_name">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 63 26pt &quot;Yu Gothic UI Semibold&quot;;
border-bottom: none;</string>
           </property>
           <property name="text">
            <string>Identifikace a verifikace osob s využitím EEG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="author">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);
border-bottom: none;
</string>
           </property>
           <property name="text">
            <string>Autor : Roland Žitný - xzitny01</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="supervisor">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);
border-bottom: none;
</string>
           </property>
           <property name="text">
            <string>Vedúci práce : Ing. Jan Tinka</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="main_function">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>60</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item>
       <widget class="QWidget" name="LoginWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
border: 1px solid #C1DBF0;
border-left: none;
border-top-right-radius: 15px;
border-bottom-right-radius: 15px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(33, 147, 176), stop:1 rgb(109, 213, 237));

}

QWidget::hover{
border: 5px groove #EFF0F1;	
border-left: none;
}</string>
        </property>
        <layout class="QVBoxLayout" name="LoginLayout" stretch="0,0,0,0,0,0,0,0">
         <item>
          <widget class="QLabel" name="LoginLabel">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Segoe MDL2 Assets&quot;;
border: none;

</string>
           </property>
           <property name="text">
            <string>Prihlásenie</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="LoginErrorLabel">
           <property name="styleSheet">
            <string notr="true">border: none;
background-color: rgba(255, 255, 255,0);
font: 10pt &quot;Arial&quot;;
color: red;
</string>
           </property>
           <property name="text">
            <string>Formulár musí byť vyplnený</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="LoginLine">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 230);
border-radius: 0px;
border: none;
</string>
           </property>
           <property name="placeholderText">
            <string>Prihlasovacie meno</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="LoginBtn">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: qradialgradient(cx:0, cy:0, radius: 1,
                fx:0.5, fy:0.5, stop:0 #0c5685 ,stop:0.5 #041f30 stop:1 #132433);

border-radius: 100px;
border-width: 5px;
padding: 2px;
border-style: groove;
border-color: #C1DBF0;
}

QPushButton::hover
{
background-color: qradialgradient(cx:0, cy:0, radius: 1.2,
                fx:0.5, fy:0.5, stop:0 #0D659D ,stop:0.5 #05263B stop:1 #192E41);
border-color: #EFF0F1;

QWidget#LoginWidget{
border: 10px;
}

}

QPushButton::pressed {
border-width: 3px;
border-color: #82807F
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../resources/icons/LoginIcon.svg</normaloff>../../resources/icons/LoginIcon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>120</width>
             <height>120</height>
            </size>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="Spacer_Log_x_Reg">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="RegistrationWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
border: 1px solid #C1DBF0;
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
border-right: none;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(253, 213, 94), stop:1 rgb(250, 208, 44));
}
QWidget::hover{
border: 5px groove #EFF0F1;
border-right: none;	
}</string>
        </property>
        <layout class="QVBoxLayout" name="RegistrationLayout">
         <property name="rightMargin">
          <number>11</number>
         </property>
         <item>
          <widget class="QLabel" name="RegLabel">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Segoe MDL2 Assets&quot;;
border: none;</string>
           </property>
           <property name="text">
            <string>Registrácia</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="RegErrorLabel">
           <property name="styleSheet">
            <string notr="true">border: none;
background-color: rgba(255, 255, 255,0);
font: 10pt &quot;Arial&quot;;
color: red;
</string>
           </property>
           <property name="text">
            <string>Formulár musí byť vyplnený</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="RegUserLogin">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 230);
border-radius: 0px;
border: none;</string>
           </property>
           <property name="placeholderText">
            <string>Prihlasovacie meno</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="RegUserName">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 230);
border-radius: 0px;
border: none;</string>
           </property>
           <property name="placeholderText">
            <string>Meno</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="RegUserSurname">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 230);
border-radius: 0px;
border: none;
</string>
           </property>
           <property name="placeholderText">
            <string>Priezvisko</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="RegistrationBtn">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: qradialgradient(cx:0, cy:0, radius: 1,
                fx:0.5, fy:0.5, stop:0 #FFB001 ,stop:0.5 #FF8000 stop:1 #DC7726);

border-radius: 100px;
border-width: 5px;
padding: 2px;
border-style: groove;
border-color: #FFED86;
}

QPushButton::hover
{
background-color: qradialgradient(cx:0, cy:0, radius: 1.2,
                fx:0.5, fy:0.5, stop:0 #ffc001 ,stop:0.5 #ff8c00 stop:1 #e37a27);
border-color: #fff2ab;
}

QPushButton::pressed {
border-width: 3px;
border-color: #F8CF2C
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../resources/icons/RegIcon.svg</normaloff>../../resources/icons/RegIcon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>120</width>
             <height>120</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
